misc/bazel/pkg.bzl

```diff
@@ -277,28 +283,36 @@ def codeql_pack(
         visibility = None,
         install_dest = "extractor-pack",
         compression_level = None,
+        arch_overrides = None,
+        prefix_override = None,
```

Comment:
nit: I would call this `zip_prefix`, and then go with `zip_prefix = zip_prefix or name` (and then write in the docstring that "`zip_prefix` is the prefix used in the zip files, (`name` by default)". I think it would be clearer that way.

Reply:
Good idea! done.